marketpulse/base/static/css/app.css

```diff
@@ -336,3 +297,57 @@ select.form-control + small.error {
         height: 270px;
     }
 }
+
+
+/* Form
+==================== */
+
+#onlineshop {
+    display: none;
+}
+
+.switch {
+    text-align: right;
+}
+
+.switch label:first-child {
+    float:left;
+    font-weight: normal;
+    margin-top: 15px;
+}
+
+.hasplan {
+    display:none;
+}
+
+.planprices {
+    border-top: 1px lightgray solid;
+}
+
+.planprices:first-child {
+    border-top: none;
+}
+
+select.form-control {
+    height: 4rem;
+    line-height: 4rem;
+}
+
+fieldset {
+    font-size:1.8rem;
+}
+
+fieldset legend {
+    background-color: #CD5F5A;
+    color: white;
+    background-image: none !important;
```

Comment:
nit: Is there any way to avoid the use of  `!important`?
http://mozweb.readthedocs.org/en/latest/reference/css-style.html